R/kernel_constructors.R

```diff
@@ -50,101 +48,137 @@ NULL
 
 #' @rdname kernels
 #' @export
-bias <- function (variance, dim = 1) {
+bias <- function (variance) {
   greta_kernel("bias",
-               gpflow_name = "Bias",
-               parameters = list(variance = variance),
-               dim = dim)
+               tf_name = "tf_bias",
+               parameters = list(variance = variance))
 }
 
 #' @rdname kernels
 #' @export
-white <- function (variance, dim = 1) {
```

Comment:
the white kernel is the equivalent of an IID Gaussian random effect (random intercept in mixed-effects modelling parlance), so the kernel is diagonal with the specified variance if it's a self-kernel, or all 0s if it's not the self-kernel

Reply:
it would be good to have this in here!

Reply:
updated to add white kernel + tests